src/main/java/ru/testtask/controller/ChatController.java

```diff
@@ -0,0 +1,52 @@
+package ru.testtask.controller;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.messaging.handler.annotation.DestinationVariable;
+import org.springframework.messaging.handler.annotation.MessageMapping;
+import org.springframework.messaging.handler.annotation.Payload;
+import org.springframework.messaging.simp.SimpMessageHeaderAccessor;
+import org.springframework.messaging.simp.SimpMessageSendingOperations;
+import org.springframework.stereotype.Controller;
+import ru.testtask.model.Message;
+import ru.testtask.service.MessageService;
+
+import java.util.List;
+
+import static java.lang.String.format;
+
+@Controller
+public class ChatController {
+    @Autowired
+    private SimpMessageSendingOperations messagingTemplate;
+
+    @Autowired
+    private MessageService messageService;
+
+    @MessageMapping("/chat/{roomId}/sendMessage")
+    public void sendMessage(@DestinationVariable String roomId, @Payload Message message) {
+        if (message.getContent().length() > 1000){
+            message.setContent(message.getContent().substring(0, 1000));
+        }
+
+        messageService.addMessage(message);
+
+        messagingTemplate.convertAndSend(format("/channel/%s", roomId), message);
+    }
+
+    @MessageMapping("/chat/{roomId}/addUser")
+    public void addUser(@DestinationVariable String roomId, @Payload Message message,
```

Comment:
Please use room name instead `roomId` to add user to room. If it is new room name, then you need to create new room. Room name should be unique.